quickstarts/add-healthchecks-quickstart.yaml

```diff
@@ -12,6 +12,19 @@ spec:
   description: You just created a sample application. Now, let’s add health checks
     to it.
   prerequisites: [You completed the "Getting started with a sample" quick start.]
+  accessReviewResources:
+  - group: apps
+    resource: daemonsets
+    verb: update
+  - group: apps
+    resource: deployments
+    verb: update
+  - group: apps.openshift.io
+    resource: deploymentconfigs
+    verb: update
+  - group: apps
+    resource: statefulsets
+    verb: update
```

Comment:
I believe this will check if the user can update these resources in all namespaces, which will hide the quick start for users who can only update these resources in certain workspaces. Have you tested with normal users?

This seems to be the case for other access reviews below as well.

Reply:
I verified this on a dev cluster, it hides the quick start for users who can only update the resources in some namespaces.